src/core/components/hv-navigator/index.tsx

```diff
@@ -199,10 +224,7 @@ export default class HvNavigator extends PureComponent<Types.Props> {
           <Stack.Navigator
             id={id}
             screenOptions={({ route }: Types.NavigatorParams) => ({
-              header: undefined,
-              headerMode: 'screen',
-              headerShown: SHOW_NAVIGATION_UI,
-              title: route.params?.url || id,
+              ...this.stackScreenOptions(route),
             })}
```

Comment:
minor: remove an iteration
```suggestion
            screenOptions={({ route }: Types.NavigatorParams) =>
              this.stackScreenOptions(route),
            }
```